src/components/signatories.js

```diff
@@ -13,7 +13,10 @@ export default function Signatories() {
     <UL>
       {signers.map(({name, jobtitle}, index) => (
         <LI key={`${name}_${index}`}>
-          {[name, jobtitle].filter(Boolean).join(', ')}
+          {[name, jobtitle]
+            .filter(Boolean)
+            .map(s => s.trim())
```

Comment:
Oops, we should actually trim before filter to deal with space-only strings. I'll fix in a later PR.

Reply:
Ah, I thought it could be `undefined` but didn't consider that `""` is also false-y. Might it be better to move the trim to `scripts/update-signers.js` so that the underlying JSON file is kept in a cleaner state?

Reply:
Hm, there's a danger of an NPE if we move the trim before the filter. Keep an eye out for that when revisiting, the last thing we want is the site to go down because of an empty form entry.

(Do we have an error boundary around this component?)